cmake/platform.cmake

```diff
@@ -31,18 +31,18 @@ message(STATUS "CMAKE_CROSSCOMPILING=${CMAKE_CROSSCOMPILING}")
 omr_detect_system_information()
 
 # Pickup OS info 
-include(cmake/platform/os/${OMR_HOST_OS}.cmake OPTIONAL)
+include(${OMR_ROOT}/cmake/platform/os/${OMR_HOST_OS}.cmake OPTIONAL)
```

Comment:
Seems no? Not sure why, but in the consuming project that inspired this work that still fails to find the include files.